tests/parts/test_slide.py

```diff
@@ -776,6 +784,13 @@ def related_fixture(self, slide_layout_, related_parts_prop_):
         getitem_.return_value.slide_layout = slide_layout_
         return slide_master_part, rId, getitem_, slide_layout_
 
+    @pytest.fixture
+    def theme_fixture(self, part_related_by_, theme_, theme_part_):
```

Comment:
That is pretty consistent throughout this codebase.  Just on this one file alone there are 3 `slide_` fixtures.  Each class of tests has whatever fixtures it needs and names and even entire fixtures get repeated.  While it has a bit of repetition, it also makes each class of tests self contained

Reply:
Ah okay, sounds good. I should have noted that I was reviewing this on github so I was reading this up and down instead of having VS code functionality to help me find the function so it took me a bit to figure out that wasn't using a fixture from another class.